test/symlink.js

```diff
@@ -268,46 +279,56 @@ describe('symlink stream', function() {
     stream.end();
   });
 
-  it('should write directories to the right folder', function(done) {
-    var inputPath = path.join(__dirname, './fixtures/wow');
-    var inputBase = path.join(__dirname, './fixtures/');
-    var expectedPath = path.join(__dirname, './out-fixtures/wow');
-    var expectedBase = path.join(__dirname, './out-fixtures');
-    var expectedMode = parseInt('677', 8) & ~process.umask();
-
-    var expectedFile = new File({
-      base: inputBase,
-      cwd: __dirname,
-      path: inputPath,
-      contents: null,
-      stat: {
-        isDirectory: function() {
-          return true;
+  [{
```

Comment:
I prefer not to have generative testing.  It allows for separate refactoring of tests